opencxl/drivers/cxl_bus_driver.py

```diff
@@ -304,7 +304,7 @@ async def _scan_pcie_dvsec_for_cxl_devices(self, device_info: CxlDeviceInfo):
             desired_interleave = (size_low >> 8) & 0b11111
             memory_active_timeout = (size_low >> 13) & 0b111
             memory_active_degraded = bool((size_low >> 16) & 0b1)
-            memory_size = (size_high << 32) | (size_low & 0xF000000)
+            memory_size = (size_high << 32) | (size_low & 0xF0000000)
```

Comment:
We need the upper four bits in the size_low for the size. The original one had a typo (one less 0 at the end).